lib/file/preallocate_unix.go

```diff
@@ -47,7 +47,6 @@ func PreAllocate(size int64, out *os.File) (err error) {
 			// Try the next flags combination
 			index++
 			atomic.StoreInt32(&fallocFlagsIndex, index)
-			fs.Debugf(nil, "preAllocate: got error on fallocate, trying combination %d/%d: %v", index, len(fallocFlags), err)
```

Comment:
I removed this for now because there was an import cycle I couldn't seem to avoid with the `cacheroot`, `file`, and `fs` packages. If there is a way to avoid this or a better place to define the `CreateCacheRoot` function please let me know.

Reply:
`lib/file` is being modified concurrently by pr https://github.com/rclone/rclone/pull/5401#issuecomment-913845588
and needs attention of @albertony @ncw

Reply:
probably we could introduce an interface for adhoc open option?